_docs/welcome/feature-stages.md

```diff
@@ -29,94 +28,75 @@ within the product, not to the product as a whole. Here is a high level descript
-| [Bring your own service name/DNS](https://docs.google.com/document/d/1If-t_g7CZnRX_dc0NG4y6CRGoI5FhkgUCqnNmPt9eFY/edit)         	   | Alpha
+| [Prometheus Integration](https://istio.io/docs/guides/telemetry.html)         | Beta
+| [Local Logging (STDIO)](https://istio.io/docs/guides/telemetry.html)               | Beta
+| [Statsd Integration](https://istio.io/docs/reference/config/mixer/adapters/statsd.html)         	   | Beta
+| [Service Dashboard in Grafana](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html)       | Beta
+| [Stackdriver Integration](https://istio.io/docs/reference/config/mixer/adapters/stackdriver.html)       | Alpha
+| [Service Graph](https://istio.io/docs/tasks/telemetry/servicegraph.html)       | Alpha
 
 
 ### Security
 
 
 | Feature           | Phase        
 |-------------------|-------------------
-| [Kubernetes Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)               | Beta
-| [Istio mTLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
+| [Deny Checker](https://istio.io/docs/reference/config/mixer/adapters/denier.html)         | Beta
+| [List Checker](https://istio.io/docs/reference/config/mixer/adapters/list.html)        | Beta
+| [Kubernetes: Service Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)               | Beta
 | [Pluggable Key/Cert Support for Istio CA](https://istio.io/docs/tasks/security/plugin-ca-cert.html)        | Beta
-| [Istio on Istio (secure istio components)](https://docs.google.com/document/d/1YzYPddihbLgJhme27-md9COn0NnKUKX_xCJ_GPXU4Fw/edit#heading=h.jbmfrt4h5lj0) | Alpha
-| [Incremental mTLS](https://docs.google.com/document/d/1D7wZCQjVB72Wlwr5ZxP5WUmn3FUDr-XzfX8OodPXe8Y/edit)    | Alpha
-| [VM Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)         | Alpha
+| [Service-to-service mutual TLS](https://istio.io/docs/concepts/security/mutual-tls.html)         | Beta
+| [Incremental Enablement of service-to-service mutual TLS](https://docs.google.com/document/d/1D7wZCQjVB72Wlwr5ZxP5WUmn3FUDr-XzfX8OodPXe8Y/edit)    | Alpha
+| [VM: Service Credential Distribution](https://istio.io/docs/concepts/security/mutual-tls.html)         | Alpha
+| [OPA Checker](https://github.com/mangchiandjjoe/istio/blob/d5390f6e436225949907d77ad3e9747a9bc26722/mixer/adapter/opa/README.md)  - **New to 0.4**      | Alpha
 
-### User Interface
 
 
-| Feature           | Phase       
-|-------------------|-------------------
-| [Grafana Dashboard](https://istio.io/docs/tasks/telemetry/using-istio-dashboard.html) | Alpha
-| [Service Graph](https://istio.io/docs/tasks/telemetry/servicegraph.html)    | Alpha
+### Core
 
 
-### Performance and Scalability
+| Feature           | Phase        
+|-------------------|-------------------
+| [Kubernetes: Istio Data Plane: Envoy Installation and Traffic Interception](https://istio.io/docs/setup/kubernetes/)        | Beta
```

Comment:
nit: Double ':' bugs me, not sure how to fix.

Maybe just drop the "Istio Data Plane:" part:

Kubernetes: Envoy Installation and Traffic Interception